src/App.js

```diff
@@ -1,7 +1,23 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+// @ts-ignore
```

Comment:
Not too sure why we're adding TypeScript syntax here 🤔 

You should go ahead and remove this